make/harbor.yml.tmpl

```diff
@@ -215,6 +215,13 @@ _version: 2.12.0
 #   # username:
 #   # sentinel_master_set must be set to support redis+sentinel
 #   #sentinel_master_set:
+#   # tls configuration for redis connection
+#   # only server-authentication is supported
+#   # mtls for redis connection is not supported
+#   # tls connection will be disable by default 
+#   tlsOptions:
+#   # if the tlsOptions.rootCA been specified, then tls connection will be enabled.
```

Comment:
If tlsOptions.rootCA has been specified, the TLS connection will be enabled.

Reply:
Thanks for the correction!